docs/source/development/traffic_ops_api/v12/user.rst

```diff
@@ -454,6 +454,11 @@ Users
   +--------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------+
   | ``checkPath``            | string | The path portion of the URL to check this deliveryservice for health.                                                                |
   +--------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------+
+  | ``deepCachingType``      |  int   | When to do Deep Caching for this Delivery Service:                                                                                   |
```

Comment:
since this an enum in the database, could we switch it to a string here?

Reply:
Yeah, I still need to update all the docs since I changed everything to use an enum yesterday.